### ISSUE TITLE
[feature] Click-through relations

### ISSUE THREAD

**Issue by heymartinadams** [prisma/studio#600]
`2.13.1`

When we click on a relation, it’s because we want to see more about that relation. The question I have, therefore, is why not just open the relation in a new tab right away? That saves the user a second click to the **Open in new tab** link (which is placed on the bottom-left — sometimes far away from the cursor on large screens).

It seems to me this could be a powerful shorthand for users, especially if combined with #569
![image](https://user-images.githubusercontent.com/11673745/103389265-ae1a4480-4ac2-11eb-82cd-c66109eb8af4.png)

**Comment by madebysid** [comment]
@heymartinadams Opening it in a new tab always would not make it possible to change the value of the relation though. I guess a `Cmd/Ctrl+Click` could open it in a new tab (mimicking browser link behaviour), but I'll let @kuldar take a look at this as well :D

**Comment by chrissisura** [comment]
Yes command clicking through your relations like in VSCode would be amazing! It would already be a huge improvement if you don't have to memorize the ids but instead go to that new Table with the cmd-clicked item selected :)

**Comment by igalklebanov** [comment]
Hey :wave:

Prisma ORM 7 introduces a brand new standalone [Studio](https://www.prisma.io/docs/orm/tools/prisma-studio), built from the ground up to work everywhere, with or without Prisma ORM. Consequently, all "old Studio" issues are being closed as there won't be any further work on it. :mega:

The new Studio is also available in the Prisma VS Code extension and in the Prisma Console application. :electric_plug:

Help shape, improve and fix the new Studio with us in this repository by submitting bug reports and feature requests. :construction_worker:
